README.md

```diff
@@ -24,12 +24,12 @@ Other examples are provided in the _examples_ [directory](./examples).
 
 ## Installation
 
-We have multiple ways to install Kompose. Our prefered method is downloading the binary from the latest GitHub release.
+We have multiple ways to install Kompose. Our preferred method is downloading the binary from the latest GitHub release.
 
 Our entire list of installation methods are located in our [installation.md](/docs/installation.md) document.
 
 Installation methods:
-  - [Binary (Prefered method)](README.md)
+  - [Binary (Preferred method)](#binary-installation)
```

Comment:
I think this should be `/docs/installation.md#binary-installation` otherwise lgtm.

Reply:
We don't have a section named `binary-installation` in `/docs/installation.md` (actual method is there though)
but the same installation method is present in `README.md` itself, so I thought linking to the section from `README.md` will be better solution.
should I link to `/docs/installation.md` instead?

Reply:
@bhavin192 sorry, link to: https://github.com/kubernetes/kompose/blob/master/docs/installation.md#github-release

so: `docs/installation.md#github-release`

Reply:
@cdrage I got it now, so the section is like a index of `installation.md` but again won't it be little bit confusing for user? as we have instructions there on the `README.md` itself (for binary installation) and we are redirecting to `installation.md` for the same instructions?
Can we have list of all methods after the Binary Installation method (`README.md`) ?